File: Pattern10.java
import java.util.Scanner;

// A 
// A B 
// A B C 
// A B C D 
// A B C D E 
// A B C D E F
 
public class Pattern10 {
    public static void main(String[] args) {
        int n;
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the number:");
        n = in.nextInt();
        int val = 64;
        for (int i = 1; i <=n; i++) {
            for(int j=1;j<=i;j++){
                System.out.print((char)(val+j)+" ");
            }
            System.out.println();
        }
    }
}

File: Pattern10.py
# A 
# A B 
# A B C 
# A B C D 
# A B C D E 
# A B C D E F

n = int(input("Enter the number:"))
val = 64
for i in range(1,n+1):
    for j in range(1,i+1):
        print(chr(ord(chr(val))+j)," ",end='')
    print()